rclpy/rclpy/topic_or_service_is_hidden.py

```diff
@@ -15,7 +15,7 @@
 HIDDEN_TOPIC_PREFIX = '_'
 
 
-def topic_or_service_is_hidden(name):
+def topic_or_service_is_hidden(name: str) -> bool:
```

Comment:
Should we remove the typing info from the param documentation below?

```diff
-:param name str: topic or service name to test
+:param name: topic or service name to test
```

Reply:
see https://github.com/ros2/rclpy/pull/1230#discussion_r1517200944